RELEASE_NOTES.md

```diff
@@ -1,5 +1,86 @@
 <!-- markdown-link-check-disable -->
 
+# 0.82.0
+
+The 0.82.0 release delivers significant improvements to [Kubernetes orchestrator](https://docs.zenml.io/stacks/stack-components/orchestrators/kubernetes), enhanced documentation, and numerous fixes to improve overall stability and performance. Key highlights include configurable max parallelism for Kubernetes orchestrator, customizable pod name prefixes and scheduler options, improved runner timeouts, and support for private service connections in Vertex AI. This release also includes comprehensive documentation updates, and library compatibility improvements for NumPy and Pandas.
+
+# Features
+
+- Added max parallelism option for [Kubernetes orchestrator](https://docs.zenml.io/stacks/stack-components/orchestrators/kubernetes)
+- Added support for pod name prefixes and scheduler configuration
+- Added private service connect option for [Vertex AI orchestrator](https://docs.zenml.io/stacks/stack-components/orchestrators/vertex)
+- Made runner timeout easily configurable
+- Added storage for list of Python packages
+
+# Improvements
+
+- Adjusted GitHub code repo regex pattern for better compatibility
+- Improved build invalidation when parent Dockerfile changes
+- Enhanced directory handling during code download
+- Added ability to list model versions without models
+- Added support for extra attributes in all ZenML models
+- Disabled default project behavior for pro workspaces
+
+# Fixes
+
+- Fixed run templates listing
+- Eliminated premature active project warning logs
+- Updated scikit-learn requirement in SklearnIntegration
+- Updated NumPy integration to work with both 1.x and 2.x library versions
+- Added Pandas custom data type error handling and logging
+- Removed unnecessary and invalid settings
+
```

Comment:
This should have an explicit section with breaking changes and how to deal with them